students/ZachCooper/session03/list_lab.py

```diff
@@ -0,0 +1,76 @@
+#!usr/bin/env python
+
+#Series 1
+
+#Create and print list
+fruit = ['Apples', 'Pears', 'Oranges', 'Peaches']
+print(fruit)
+#User input ask for new fruit
+new_fruit = input("Please tell me another type of fruit that is not in the list")
+#Append list with user fruit
+fruit.append(new_fruit)
+print(fruit)
+#Ask the user for a number of listed fruit indexes
+num_question = int(input("Please give me a number from 0-4:"))
+#Return the number back and fruit name at that index
+print(fruit[num_question])
+#Add another fruit to the list using "+"
+print(fruit + ['Melons'])
+#Add another fruit to list using insert(), then display
+fruit.insert(0, 'Strawberries')
+print(fruit)
+#Display fruits that begin with "P", using a for loop
+for x in fruit:
+	if x == 'P':
+		print(x)
```

Comment:
indentation still wrong -- yo've got those pesky <tab>s in there.

Let's try to get your editor configured before class.